<?xml version="1.0" ?>
<launch>        
    <!-- UI -->
    <rosparam file="$(find mh5_ui)/config/menu.yaml" command="load"/>
    <node pkg="mh5_ui" type="mh5_ui_node.py" name="ui" output="screen">
        <param name="rate" value="10" />
        <param name="torque_service" value="torque_control/switch_torque" />
        <param name="reboot_service" value="torque_control/reboot" />
    </node>
</launch>